rasa_sdk/executor.py

```diff
@@ -188,7 +188,17 @@ def _import_submodules(
         :rtype: dict[str, types.ModuleType]
```

Comment:
there's another usage of this method in `register_package()` where an `ImportError` is excepted